routes/v0/post-surveys.js

```diff
@@ -0,0 +1,39 @@
+const models = require('../../models');
+const { SafeRouter, publicEndpoint } = require('../../permissions');
+
+const { PostSurvey } = models;
+const PostSurveyController = require('../../controllers/post-survey-controller')(PostSurvey);
+
+const postSurveysRouter = SafeRouter();
+
+postSurveysRouter.get('/', publicEndpoint, async (
+  /** @type {import('express').Request} */ req,
+  /** @type {import('express').Response} */ res,
+) => {
+  const { accountSid } = req;
+  const postSurveys = await PostSurveyController.getPostSurveys(req.query, accountSid);
+  res.json(postSurveys);
+});
+
+postSurveysRouter.post('/', publicEndpoint, async (
```

Comment:
It feels weird to make this look like it's a `publicEndpoint`, meaning that it's open to everyone, even though it's really not.  You do need an account-specific key.  The comment on `publicEndpoint` says, "A middleware that just marks an endpoint as open".  Can we move some things around or rename some things to make it clear that this isn't fully public and requires a key?